src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -213,7 +213,7 @@ package com.videojs.providers{
         }
         
         public function get playing():Boolean{
-            return _isPlaying;
+            return _playbackStarted;
```

Comment:
Got it. The `playing()` function is pretty poorly named too then. Thanks for clearing that up.